store.html

```diff
@@ -0,0 +1,153 @@
+      type="text/css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="stylesheet" href="/css/store.css" />
+    <link rel="shortcut icon" type="image/xicon" href="favicon.ico" />
+  </head>
+  <body>
+    <header>
+      <a class="logo" href="index.html"
+        ><img src="/img/karmalogo.svg" alt="karma logo"
+      /></a>
+      <nav>
+        <ul class="navlist">
+          <input type="checkbox" id="check" />
+          <div class="menu">
+            <li class="navitems">
+              <a class="navlinks" href="index.html">Meet Karma</a>
+            </li>
+            <li class="navitems">
+              <a class="navlinks" href="#">How it Works</a>
+            </li>
+            <li class="navitems">
+              <a class="navlinks" href="store.html">Store</a>
+            </li>
+            <li class="navitems"><a class="navlinks" href="#">Blog</a></li>
+            <li class="navitems"><a class="navlinks" href="#">Help</a></li>
+            <li class="navitems"><a class="navlinks" href="#">Login</a></li>
+          </div>
+          <label for="check" class="openmenu"
+            ><img src="/level3/menuhamburger.svg" alt="burger"
+          /></label>
+        </ul>
+      </nav>
+    </header>
+    <main class="storemain">
+      <section class="formsection">
+        <h1>Order your Karma wifi device today!</h1>
+
+        <form action="#">
+          <div class="fieldset">
+            <div class="flex">
+              <label for="firstname">First name *</label>
+              <input type="text" id="firstname" required />
+            </div>
+            <div class="flex">
+              <label for="lastname">Last name *</label>
+              <input type="text" id="lastname" required />
+            </div>
+          </div>
+
+          <div class="address">
+            <div class="flex">
+              <label for="addressone">Address *</label>
+              <input type="text" id="addressone" required />
+            </div>
+          </div>
+
+          <div class="address">
+            <div class="flex">
+              <label for="addresstwo">Address 2</label>
+              <input type="text" id="addresstwo" />
+            </div>
+          </div>
+
+          <div class="fieldset">
+            <div class="flex">
+              <label for="city">City *</label>
+              <select name="city" id="city" required>
+                <option value="" disabled selected>Select your city...</option>
+                <option value="tehran">Tehran</option>
+                <option value="london">London</option>
+                <option value="cardiff">Cardiff</option>
+              </select>
+            </div>
+
+            <div class="flex">
+              <label for="postcode">Postcode *</label>
+              <input type="text" id="postcode" required />
+            </div>
+          </div>
+
+          <div class="radiobox">
```

Comment:
This should be a fieldset tag instead of a div, because it's a group of related form controls (radio buttons)